src/device-registry/models/Site.js

```
@@ -360,60 +362,73 @@
-    const error = new Error("Duplicate values found in grids array.");
-    return next(error);
-  }
+siteSchema.pre(
+  ["updateOne", "findOneAndUpdate", "updateMany", "update", "save"],
+  function(next) {
+    // Handle updates
+    if (this.getUpdate) {
+      const updates = this.getUpdate();
+      if (updates) {
+        // Remove latitude and longitude from updates
+        if (updates.latitude) delete updates.latitude;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)